README.md

```diff
@@ -19,12 +19,12 @@ You will need the following things properly installed on your computer.
 5. This API relies on a Postgresql database for its data. At this point in the set up, you must decide if you want to use a local instance of the database or if you want to connect to the remote database used by our development environment. If you're only making changes to the API code, you will likely be fine using the development environment database. If you're making changes to the scripts in `/migrations` or performing database updates, you should start with a local database so that you can work on your changes without affecting any remote environments shared across the team. If you're unsure which approach you should take, ask someone on the team for help.
 
-  ### Setting up a local database instance with `docker-compose`
+  ### Setting up a local database instance with docker
   You can set up a local instance of the database that this API relies on using Docker. This is helpful for testing changes to the API locally and for making updates to the "migration" scripts themselves. To set up a local instance of the database:
   * Makes sure you have [docker installed](https://www.docker.com/get-started/). If you don't, consider installing it via [homebrew](https://www.brew.sh).
-  * Once you have docker installed, try running `docker-compose up` from the root directory of this repository. The file `docker-compose.yml` should spin up a container running the default Postgresql image. If this step is successful, you should see logs indicating that the database is running and ready to accept connections in your terminal. You will need to keep this container running while running the project.
+  * Once you have docker installed, try running `docker compose up database` from the root directory of this repository. The file `compose.yaml` should spin up a container running the default Postgresql image. If this step is successful, you should see logs indicating that the database is running and ready to accept connections in your terminal. You will need to keep this container running while running the project.
   * Make sure you have locally installed [jq](https://stedolan.github.io/jq/download/) and [Postgres](https://www.postgresql.org/download/). Tip: `brew install jq` and `brew install postgresql` for Mac users using Homebrew.
-  * Double check that you created your `.env` file and copied over the contents of `sample.env`. The contents of `sample.env` are pre-configured to connect to the database created by `docker-compose.yml`
+  * Double check that you created your `.env` file and copied over the contents of `sample.env`. The contents of `sample.env` are pre-configured to connect to the database created by `docker compose.yml`
```

Comment:
Did you find some documentation that led you to rename `docker-compose.yml` to `compose.yml`? I ask because in my experience, `docker-compose.yml` is the built in "default" file name that docker compose looks for when you run the cli commands without specifying a file name. I think we should stick to those sorts of defaults when we can so that repos are more familiar to engineers with experience with these tools.